proto/stream_plan.proto

```diff
@@ -261,11 +261,24 @@ message GroupTopNNode {
   bool with_ties = 6;
 }
 
+message DeltaExpression {
+  expr.ExprNode.Type delta_type = 1;
+  expr.ExprNode delta = 2;
+}
+
+message InequalityPair {
+  uint32 key_required_larger = 1;
+  uint32 key_required_smaller = 2;
+  bool generate_watermark = 3;
```

Comment:
Please add som comments here, thanks!

Reply:
> Please add som comments here, thanks!

ok let me copy comments in `src/frontend/src/expr/mod.rs` here.